components/desktop/remmina/Makefile

```diff
@@ -0,0 +1,60 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2020 Marco van Wieringen
+#
+
+BUILD_BITS= 64
+BUILD_STYLE= cmake
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		Remmina
+COMPONENT_VERSION=	1.4.7
+COMPONENT_COMMIT=	da3b4196eabbc5ad9719bd35fa6a633f4aa6337e
+COMPONENT_SUMMARY=	Remmina remote desktop client
+COMPONENT_PROJECT_URL=	https://www.freerdp.com
+COMPONENT_FMRI=		desktop/remote-desktop/remmina
+COMPONENT_CLASSIFICATION=	Applications/Internet
+COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_COMMIT)
+COMPONENT_ARCHIVE=	$(COMPONENT_SRC).zip
+COMPONENT_ARCHIVE_URL=	https://github.com/FreeRDP/Remmina/archive/$(COMPONENT_COMMIT).zip
+COMPONENT_ARCHIVE_HASH=	\
```

Comment:
There is a tagged release on github for Remmina 1.4.7:
https://github.com/FreeRDP/Remmina/archive/v1.4.7.tar.gz
Please use this in COMPONENT_ARCHIVE_URL (construct with parts of this path replaced by the appropriate COMPONENT_* variables)

Reply:
Ok done.